src/utils_project.js

```diff
@@ -181,6 +164,71 @@ go.utils_project = {
         };
     },
 
+    // taking identity_uuid and quiz_uuid, returns tracker_uuid
+    init_tracker: function(im, identity_id, quiz_id) {
+        var payload = {
+            "identity": identity_id,
+            "quiz": quiz_id
+        };
+
+        return go.utils
+            .service_api_call("continuous-learning", "post", null, payload, 'tracker/', im)
+            .then(function(json_post_response) {
+                return json_post_response.data.tracker_id;
+        });
+    },
+
+    log_quiz_answer: function(im, quiz_question, answer_value, answer_text, answer_correct, response, tracker_id) {
+        var payload = {
+            "question": quiz_question.id,
+            "question_text": quiz_question.question,
+            "answer_value": answer_value,
+            "answer_text": answer_text,
+            "answer_correct": answer_correct,
+            "response_sent": response,
+            "tracker": tracker_id
+        };
+
+        return go.utils
+            .service_api_call("continuous-learning", "post", null, payload, 'answer/', im)
+            .then(function(json_post_response) {
+                return json_post_response.data;
+        });
+    },
+
+    close_tracker: function(im, tracker_id) {
+        var endpoint = "tracker/"+tracker_id+"/";
+        var payload = {
+            "complete": "True",
+            "completed_at": go.utils.get_today(im.config)
```

Comment:
this helper is not granular enough, it's just today not now (including timestamp)